pkg/globalnet/controllers/ipam/ipam.go

```diff
@@ -335,13 +343,10 @@ func (i *Controller) handleRemovedPod(obj interface{}) {
 	}
 }
 
-func (i *Controller) annotateGlobalIp(key string, annotations map[string]string) (map[string]string, error) {
-	if annotations == nil {
-		annotations = map[string]string{}
-	}
+func (i *Controller) annotateGlobalIp(key string, globalIp string) (map[string]string, error) {
+	annotations := map[string]string{}
```

Comment:
Earlier, we were passing existing annotations and initializing this only if they were not present. With this change, if there are any other annotations, those would be lost.

Reply:
Done